doc/source/whatsnew/v1.0.0.rst

```diff
@@ -46,7 +46,7 @@ Other API changes
 ^^^^^^^^^^^^^^^^^
 
 - :meth:`pandas.api.types.infer_dtype` will now return "integer-na" for integer and ``np.nan`` mix (:issue:`27283`)
--
+- Added :meth:`pandas.core.frame.DataFrame.value_counts` (:issue:`5377`).
```

Comment:
needs an entry in the doc/source/reference rst to make this render. i am surprised this doesn’t show a warning

Reply:
move this to the Other Enhancements section